concent_api/core/transfer_operations.py

```diff
@@ -62,27 +64,30 @@ def store_pending_message(
     subtask: Optional[Subtask]=None,
     payment_message: Optional[message.concents.ForcePaymentCommitted]=None,
 ) -> None:
-    client          = Client.objects.get_or_create_full_clean(client_public_key)
-    receive_queue   = PendingResponse(
-        response_type   = response_type.name,
-        client          = client,
-        queue           = queue.name,
-        subtask         = subtask,
-    )
-    receive_queue.full_clean()
-    receive_queue.save()
-    if payment_message is not None:
-        payment_committed_message = PaymentInfo(
-            payment_ts=parse_timestamp_to_utc_datetime(payment_message.payment_ts),
-            task_owner_key=payment_message.task_owner_key,
-            provider_eth_account=payment_message.provider_eth_account,
-            amount_paid=payment_message.amount_paid,
-            recipient_type=payment_message.recipient_type.name,  # pylint: disable=no-member
-            amount_pending=payment_message.amount_pending,
-            pending_response=receive_queue
+    try:
```

Comment:
is it really needed here? I thought that creating `Subtask` might be a problem...

Reply:
The task is not only for subtasks, @cameel wrote about other objects as well. I checked the code and generally other (not subtask) objects that may fail with IntegrityError are in this place.